execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -431,13 +446,12 @@ impl TransactionGenerator {
                 Arc::new(AtomicUsize::new(0)),
                 |(sender_idx, new_account), account_cache| {
                     let sender = &account_cache.accounts[sender_idx];
-                    let txn = sender.sign_with_transaction_builder(
-                        self.transaction_factory
-                            .implicitly_create_user_account_and_transfer(
-                                new_account.public_key(),
-                                init_account_balance,
-                            ),
+                    let payload = aptos_stdlib::aptos_account_transfer(
```

Comment:
Does this use FA or the old coin standard?

I am wondering if you're also accidentally switching from `Coin` to `FungibleAsset` which may be slower & be confused for a slower keyless performance.